src/components/data-flash/DataFlash.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div class="data-flash">
+    <div
+      v-if="!suportDataflash"
```

Comment:
`suport` is misspelled in this PR. Please also update other occurrences.
```suggestion
      v-if="!supportDataflash"
```